src/data/teams.js

```diff
@@ -3,193 +3,193 @@ export const Teams = {
         "flagCode": "au",
-        "group": "D"
-    },
-    "BEL": {
-        "teamId": "BEL",
-        "teamName": "Belgium",
-        "flagCode": "be",
-        "group": "F"
+        "group": "B"
     },
     "BRA": {
         "teamId": "BRA",
         "teamName": "Brazil",
         "flagCode": "br",
-        "group": "G"
-    },
-    "CMR": {
-        "teamId": "CMR",
-        "teamName": "Cameroon",
-        "flagCode": "cm",
-        "group": "G"
+        "group": "F"
     },
     "CAN": {
         "teamId": "CAN",
         "teamName": "Canada",
         "flagCode": "ca",
-        "group": "F"
+        "group": "B"
+    },
+    "CHN": {
+        "teamId": "CHN",
+        "teamName": "China",
+        "flagCode": "cn",
+        "group": "D"
+    },
+    "COL": {
+        "teamId": "COL",
+        "teamName": "Colombia",
+        "flagCode": "co",
+        "group": "H"
     },
     "CRC": {
         "teamId": "CRC",
         "teamName": "Costa Rica",
         "flagCode": "cr",
-        "group": "E"
-    },
-    "CRO": {
-        "teamId": "CRO",
-        "teamName": "Croatia",
-        "flagCode": "hr",
-        "group": "F"
+        "group": "C"
     },
     "DEN": {
         "teamId": "DEN",
         "teamName": "Denmark",
         "flagCode": "dk",
         "group": "D"
     },
-    "ECU": {
-        "teamId": "ECU",
-        "teamName": "Ecuador",
-        "flagCode": "ec",
-        "group": "A"
-    },
     "ENG": {
         "teamId": "ENG",
         "teamName": "England",
         "flagCode": "gb-eng",
-        "group": "B"
+        "group": "D"
+    },
+    "ESP": {
+        "teamId": "ESP",
+        "teamName": "Spain",
+        "flagCode": "es",
+        "group": "C"
     },
     "FRA": {
         "teamId": "FRA",
         "teamName": "France",
         "flagCode": "fr",
-        "group": "D"
+        "group": "F"
     },
     "GER": {
         "teamId": "GER",
         "teamName": "Germany",
         "flagCode": "de",
-        "group": "E"
-    },
-    "GHA": {
-        "teamId": "GHA",
-        "teamName": "Ghana",
-        "flagCode": "gh",
         "group": "H"
     },
-    "IRN": {
-        "teamId": "IRN",
-        "teamName": "IR Iran",
-        "flagCode": "ir",
+    "HAI": {
+        "teamId": "HAI",
+        "teamName": "Haiti",
+        "flagCode": "ht",
+        "group": "D"
+    },
+    "IRL": {
+        "teamId": "IRL",
+        "teamName": "Ireland",
+        "flagCode": "ie",
         "group": "B"
     },
+    "ITA": {
+        "teamId": "ITA",
+        "teamName": "Italy",
+        "flagCode": "it",
+        "group": "G"
+    },
+    "JAM": {
+        "teamId": "JAM",
+        "teamName": "Jamaica",
+        "flagCode": "jm",
+        "group": "F"
+    },
     "JPN": {
         "teamId": "JPN",
         "teamName": "Japan",
         "flagCode": "jp",
-        "group": "E"
+        "group": "C"
     },
     "KOR": {
         "teamId": "KOR",
-        "teamName": "Korea Republic",
+        "teamName": "Korea",
```

Comment:
Let's make this "South Korea" - I left it in the backend as "Korea Republic", but "South Korea" seems better to me.